inappmessaging/src/main/res/values/styles.xml

```diff
@@ -16,24 +16,19 @@
     <!--Default background as white.-->
     <item name="android:src">@drawable/close_button_white_background</item>
     <item name="android:layout_gravity">end</item>
-    <item name="android:layout_marginBottom">@dimen/modal_close_button_margin_bottom</item>
```

Comment:
Please move the button on the right edge of the modal view.

Reply:
Done